lib/core/datatable/components/datatable/location-cell.component.ts

```diff
@@ -15,46 +15,56 @@
  * limitations under the License.
  */
 
-import { ChangeDetectionStrategy, Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    OnInit,
+    ViewEncapsulation
+} from '@angular/core';
 import { PathInfoEntity } from '@alfresco/js-api';
 import { DataTableCellComponent } from './datatable-cell.component';
+import { AlfrescoApiService } from '../../../services/alfresco-api.service';
 
 @Component({
     selector: 'adf-location-cell',
     changeDetection: ChangeDetectionStrategy.OnPush,
     template: `
         <ng-container>
             <a href="" [title]="tooltip" [routerLink]="link">
-                {{ displayText }}
+                {{ value$ | async }}
             </a>
         </ng-container>
     `,
     encapsulation: ViewEncapsulation.None,
     host: { class: 'adf-location-cell' }
 })
-export class LocationCellComponent extends DataTableCellComponent implements OnInit {
-
+export class LocationCellComponent extends DataTableCellComponent
+    implements OnInit {
     @Input()
     link: any[];
 
-    @Input()
-    displayText: string = '';
+    constructor(apiService: AlfrescoApiService) {
```

Comment:
would it better as we discuss rename apiService to alfrescoApiService? it helps consistency